src/components/Button/button.types.d.ts

```diff
@@ -0,0 +1,10 @@
+import { HTMLAttributes } from 'react';
+
+/**
+ * Types for Button
+ * @param {styleType} string - set or secondary primary style of the button
+ */
+
+export interface IButtonProps extends HTMLAttributes<HTMLButtonElement> {
+  styleType?: 'primary' | 'secondary';
```

Comment:
I think that `styleType` is a bad naming for the prop as in the future the behavior of the btn can change, not only its styles. So refactor it to be just `type`.